PokeAlarm/Filters/BaseFilter.py

```diff
@@ -180,7 +180,10 @@ def __init__(self, limit, eval_func, attr_name):
 
     def __call__(self, filtr, event):
         value = getattr(event, self._attr_name)  # event.event_attr
-        if Unknown.is_(value):
+        if (type(value) == list):
+            if Unknown.is_(*value):
```

Comment:
Explain what is going on with this new section of code. What are these nested if statements doing or not doing? Has this been tested, if so, how, can you show the test results and what was required to produce them?

Reply:
Since now, all the evaluated values were a single integer and this one was given to `Unknown.is_()`.
The new value `types` is a list of 1 or 2 int corresponding to the types_ids of the monster.
The new change passes the list as multiple args to `Unknown.is_()` using the `*list` syntax.